executor/point_get.go

```diff
@@ -464,10 +466,70 @@ func getColInfoByID(tbl *model.TableInfo, colID int64) *model.ColumnInfo {
 	return nil
 }
 
+type insertRuntimeStat struct {
```

Comment:
How about move `insertRuntimeStat` definition and methods into `insert_common.go` file?